alpha/veneer/basic/basic.go

```diff
@@ -52,8 +29,7 @@ func (v Veneer) renderFile(ctx context.Context, ref string) (*declcfg.Declarativ
 
 	for _, b := range cfg.Bundles {
 		if !isBundleVeneer(&b) {
-			outb = append(outb, b)
-			continue
+			return nil, fmt.Errorf("bundle veneers must define only a schema and an image; all other fields must not be defined")
```

Comment:
I think I would just say something more generic, something like: "unexpected fields present in basic veneer bundle"

"all other fields must not be defined" gives the impression that there _are_ other fields that should be left empty. In my mind, that's probably the wrong message. Basic veneer is its own format that just happens to look a bit like FBC.

Reply:
I say this because it gives us more flexibility in the future if we define the basic veneer input file separately from FBC. The two formats don't necessarily have to evolve in lock-step.

Reply:
I lean heavily toward getting the basic veneer on its own schema, so we could do `opm alpha render-veneer` instead of `opm alpha render-veneer [basic|semver]`.